src/base/search/searchpluginmanager.cpp

```diff
@@ -555,7 +555,7 @@ PluginVersion SearchPluginManager::getPluginVersion(QString filePath)
     PluginVersion version;
     while (!plugin.atEnd()) {
         QByteArray line = plugin.readLine();
-        if (line.startsWith("#VERSION: ")) {
+        if (line.startsWith("#VERSION: ") || line.startsWith("# VERSION: ")) {
```

Comment:
I am not familiar with all these Q* Objects at all, so I it would take me some time.

Honestly I don't think this function will see much extension in the future and my code is readable, so I would be happy if it could stay like this-.

Reply:
Here is my implementation, also clean it up a bit...
```c++
#include <QRegularExpression>

    while (!plugin.atEnd()) {
        const QString line = plugin.readLine().trimmed();
        const QRegularExpressionMatch match = QRegularExpression("#\\s*version\\s*:\\s*([0-9.]+)", QRegularExpression::CaseInsensitiveOption).match(line);
        if (!match.hasMatch()) continue;

        const QString versionStr = match.captured(1);
        version = PluginVersion::tryParse(versionStr, invalidVersion);
        if (version == invalidVersion) {
            LogMsg(tr("Search plugin '%1' contains invalid version string ('%2')")
                .arg(Utils::Fs::fileName(filePath), line), Log::MsgType::WARNING);
        }

        break;
    }
```

Or if you don't mind, I can push it directly to your git branch.

Reply:
heh, I didn't know you had to double back slash.

But yeah sure push it to my branch. But my useless commit would still be on it

Reply:
Ok, done, I also reword the commit message to be compliant with our coding guidelines, hope that is ok for you. And you still retain the authorship of the commit.